Packs/Workday/Integrations/Workday/Workday.yml

```diff
@@ -14,12 +14,20 @@ configuration:
   type: 9
 - display: Tenant name
   name: tenant_name
-  required: true
+  required: false
   type: 4
+  hidden: true
 - display: Token
   name: token
-  required: true
+  required: false
   type: 4
+  hidden: true
+- display: Tenant name
+  name: cred_tenant_name
+  required: false
+  type: 9
+  displaypassword: token
+  hiddenusername: true
```

Comment:
Why hiddenusername: true? the identifier here is tenant_name

Reply:
fixed